sources/Outputs/src/Job.cpp

```diff
@@ -119,6 +119,18 @@ Job::writeOutputs() {
   appender->setFilePath("dynawo.log");
   appender->setLvlFilter(def_.dynawoLogLevel);
   log->addAppenderEntry(appender);
+
+  appender = job::AppenderEntryFactory::newInstance();
+  appender->setTag("COMPILE");
```

Comment:
I don't think we should dump those files by default

Reply:
I put them because I rememeber that for the first test I performed with DFL, dynawo log was not enought. But maybe it was just the modeler log that is required

Reply:
Both of them are debug logs, so I am not sure we should use them by default. Maybe we could add them with a user option? @marcochiaramello any input on this?

Reply:
And COMPILE is only here to get logs if you use models compiled on the fly by dynawo whihch should never happen for dynaflo as we will only use black box models to speed up the simulation

Reply:
what about MODELER log ?

Reply:
MODELER is really a debug log that provides the index of each model for example

Reply:
OK, I remove it as well